src/easymotion/easymotion.ts

```diff
@@ -131,33 +131,35 @@ export class EasyMotion {
   private static getSvgDataUri(code: string, backgroundColor: string, fontFamily: string, fontColor: string,
     fontSize: string, fontWeight: string): vscode.Uri {
 
-      // Clear cache if the backgroundColor has changed
-      if (this.setBackgroundColor !== backgroundColor) {
-        this.svgCache = {};
-        this.setBackgroundColor = backgroundColor;
-      }
+    // Clear cache if the backgroundColor has changed
+    if (this.setBackgroundColor !== backgroundColor) {
```

Comment:
yeah I noticed that but didn't realize it was b/c of cache, I'll see if I can fix it